File: app/src/main/java/com/rtukpe/fixtures/ui/fixtures/FixturesMvpView.java
package com.rtukpe.fixtures.ui.fixtures;

import com.rtukpe.fixtures.data.model.Fixture;
import com.rtukpe.fixtures.ui.base.MvpView;

import java.util.ArrayList;

public interface FixturesMvpView extends MvpView {

    void updateFixtures(ArrayList<Fixture> fixtures);

}

File: app/src/main/java/com/rtukpe/fixtures/ui/fixtures/FixturesMvpView.kt
package com.rtukpe.fixtures.ui.fixtures

import com.rtukpe.fixtures.data.model.Fixture
import com.rtukpe.fixtures.ui.base.MvpView
import java.util.*

interface FixturesMvpView : MvpView {

    fun updateFixtures(fixtures: ArrayList<Fixture>)

}
